index.css

```diff
@@ -0,0 +1,136 @@
+@import url('https://fonts.googleapis.com/css2?family=Commissioner:wght@400;500;700&display=swap');
+
+*,*::before,*::after{
```

Comment:
What are trying to achieve here

Reply:
In the first line I was trying to import the fonts needed for the project...

In the third line I was trying box-size the everything so it doesn't overflow...(heard its a good practice)